examples/image-to-text/run_pipeline.py

```diff
@@ -149,6 +155,7 @@ def main():
         "hpu_graphs": args.use_hpu_graphs,
         "max_new_tokens": args.max_new_tokens,
         "ignore_eos": args.ignore_eos,
+        "use_flash_attention": args.use_flash_attention,
```

Comment:
need to pass args.causal_attention_mask, too

Reply:
Vision model already have parameter for same